src/utils/hooks/useStorage.ts

```diff
@@ -0,0 +1,43 @@
+import { getStorageItem, setStorageItem, StorageType } from '../storage';
+import { useSwr } from './useSwr';
+
+type AminoLocalStorageKey = 'current-schema';
```

Comment:
I think we should prefix with `amino:` maybe. Similar to what graphiql does for their storage keys.